### PR TITLE
Update ipv6-support.md

### PR DESCRIPTION
**Pull request by msbemba**
Added MS Teams to the document

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2693

File: microsoft-365/enterprise/ipv6-support.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ If the program that you use to connect to SharePoint Online supports IPv6, it wi
 ### Skype for Business and IPv6
 
 Please be aware IPv6 is not supported in Skype for Business and can no longer be enabled.
+
+### MS Teams and IPV6
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ipv6-support.md]:
```suggestion
### Microsoft Teams and IPV6
```

File: microsoft-365/enterprise/ipv6-support.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ If the program that you use to connect to SharePoint Online supports IPv6, it wi
...
+
+MS Teams Support IPV6. If you want to control communications to MS teams, use the IP address ranges in  [Office 365 URLs and IP address ranges](urls-and-ip-address-ranges.md
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ipv6-support.md]:
```suggestion
Microsoft Teams supports IPV6. If you want to control communications to Microsoft teams, use the IP address ranges in [Office 365 URLs and IP address ranges](urls-and-ip-address-ranges.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by rdekramer** [comment]
Thanks! To refine it further: could we add that Direct Routing only supports IPv4? The teams service and client supports both ipv4 and ipv6.

File: microsoft-365/enterprise/ipv6-support.md (hunk 2 above)
Comment (msbemba) [reply microsoft-365/enterprise/ipv6-support.md]: Making some more changes.. Will request copy review again

File: microsoft-365/enterprise/ipv6-support.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/enterprise/ipv6-support.md]: Made the change

**Comment by msbemba** [comment]
@JohanFreelancer9  Can you copy review?

File: microsoft-365/enterprise/ipv6-support.md [hunk 3]
```diff
@@ -45,6 +45,10 @@ If the program that you use to connect to SharePoint Online supports IPv6, it wi
 ### Skype for Business and IPv6
 
 Please be aware IPv6 is not supported in Skype for Business and can no longer be enabled.
+
+### Microsoft Teams and IPV6
+
+MS Teams Direct Routing only supports IPv4. MS Teams service and client supports both ipv4 and ipv6. If you want to control communications to MS Teams, use the IP address ranges in  [Office 365 URLs and IP address ranges](urls-and-ip-address-ranges.md
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/ipv6-support.md]:
```suggestion
Microsoft Teams Direct Routing only supports IPv4. The Microsoft Teams service and client support both IPv4 and IPv6. If you want to control communications to Microsoft Teams, use the IP address ranges in [Office 365 URLs and IP address ranges](urls-and-ip-address-ranges.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder, please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft  Gentle reminder, Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@kelleyvice-msft This is pending almost for a month now, Please merge this PR. Thanks.

**Comment by kelleyvice-msft** [comment]
Apologies. Notifications for this PR were being inadvertently filtered into a subfolder.

**Pull request merged**